### ISSUE TITLE
OBSOLETE: GO:0043038 amino acid activation GO:0043039    tRNA aminoacylation GO:0043041    amino acid activation for nonribosomal peptide biosynthetic process

### ISSUE THREAD

**Issue by Antonialock** [geneontology/go-ontology#15375]
should 

GO:0006426 glycyl-tRNA aminoacylation &
GO:0006544 glycine metabolic process

be related?

**Comment by hdrabkin** [comment]
No I don't think so.  We don't do this for the other tRNA aminoacylations. It's the old question of where we draw the line for substrates. Aminoacyl-trna aminoacylation are also not related to ATP metabolic process. 
That being said, you can work your way up to cellular amino acid metabolism.

**Comment by Antonialock** [comment]
ok cheers

**Comment by ValWood** [comment]
I think this is a bit strange though.

If "cellular amino acid metabolism" is a parent, then it must be logically correct to have the individual amino acid parents?

Although I had been wondering  during "Matrix" work if it was strictly correct to describe "aminoacylation" and "amino acid metabolism"

**Comment by hdrabkin** [comment]
There is a slippery slope about this: do we want to describe everything that an amino acid is involved in as an amino acid metabolism. In this case the amino acid is not really catabolized; (the breakdown of complex molecules in living organisms to form simpler ones, together with the release of energy; destructive metabolism)
GO metabolism "The chemical reactions and pathways, including anabolism and catabolism, by which living organisms transform chemical substances." 
Anabolism: the synthesis of complex molecules in living organisms from simpler ones together with the storage of energy; constructive metabolism. Protein synthesis kind of fits here;  I'm not sure if people generally think of the role of storage of energy looking at a protein, protein is generally not used as an energy source unless the body has no other option.
Not sure what you mean by 
if it was strictly correct to describe "aminoacylation" and "amino acid metabolism"
DId you mean as' amino acid metabolism'??

**Comment by deustp01** [comment]
Jumping in here in favor of narrowly drawn process boundaries, with Harold: translation is not a kind of amino acid metabolism, and if tRNA aminoacylation belongs anywhere, I'd think it belongs there. That's also an argument, responding to Val's comment from two days ago, that tRNA aminoacylation should not be a child of cellular amino acid metabolism.

Here's where terms like "causally upstream of" stop being euphemisms for "we don't exactly know what's going on" and have real logical power. Absent amino acid synthesis or uptake mediated by process one, aminoacylation cannot happen. The two distinct, non-overlapping processes are related in a precise way.

**Comment by ValWood** [comment]
> There is a slippery slope about this: do we want to describe everything that an amino acid is involved in as an amino acid metabolism. In this case the amino acid is not really catabolized; (the breakdown of complex molecules in living organisms to form simpler ones, together with the release of energy; destructive metabolism)

@hdrabkin  I think we are agreeing. At present we descirbe aminoacylation  as "amino acid metabolism". It has dual parentage to "amino acid metabolism" and translation"
I don't think we should do this?

![image](https://user-images.githubusercontent.com/7359272/38237345-eabce89e-371e-11e8-8ed2-4011ae05e1be.png)

The "matrix" intersection between "amino acid metabolism" and "translation" is purely from "aminoacyl tRNA synthases". I think these should be annotated to "translation" only?


![matrix](https://user-images.githubusercontent.com/7359272/38237664-b61870d0-371f-11e8-82e5-eeb64ed905a6.jpg)

(to interpret this, the same terms are across the top, I only have the y-axis labelled)

I think this is what @deustp01  is saying too?

so action is to remove "amino acid activation" as a  child of "amino acid metabolic process".

(I was only pointing out that in the current arrangement,  the additions
 @Antonialock  would be logically correct, and if the current arrangement persists they *should* be added)

**Comment by deustp01** [comment]
Yes, that's what I'm saying and so the removed parentage sounds right.

**Comment by ValWood** [comment]
I changed the title to reflect the action.

**Comment by hdrabkin** [comment]
The aminaocylation of tRNAs is not always a part of translation. That is why we have the term aminoacylation involved in translation. 
For example
aminoacyl-tRNAS involved in post-translational amino acid transferases
glycyl-tRNA involved in some cell wall biosynthesis.
So, along those lines, the aminoacyl-activation process does not always yield a tRNA that is involved in translation. However, just looking at your diagram, does that leave amino acid activation with no parent? I'm wondering if maybe a has_part relationship (that is SOME amino acid activation is involved in translation, but not all;

**Comment by hdrabkin** [comment]
Wait; was reading the diagram in the wrong direction; COFFEE
so "so action is to remove "amino acid activation" as a child of "amino acid metabolic process"."

**Comment by ValWood** [comment]
amino acid activation is defined
The modification of an amino acid to an active form, for incorporation into a peptide, protein or other macromolecule.

so it seems to be the same as
GO:0006418    tRNA aminoacylation for protein translation

and will cause TPV's for the non peptide synthesis aminoacylation events?

**Comment by hdrabkin** [comment]
Painfully searching for any evidence of aminoacyl-tRNA synthetase independent amino acid activation.Can't find....

**Comment by hdrabkin** [comment]
Note that tRNA aminoacylation would be the term to merge amino acid activation into.

**Comment by ValWood** [comment]
yep.

**Comment by hdrabkin** [comment]
Found;  not all aminoacid activation is tRNA dependent.
aminoacyl-CoAs
 FEBS Lett. 2011 Mar 23;585(6):893-8. doi: 10.1016/j.febslet.2011.02.018. Epub
2011 Feb 18. Aminoacyl-coenzyme A synthesis catalyzed by a CoA ligase from Penicillium
chrysogenum. Koetsier MJ(1), Jekel PA, Wijma HJ, Bovenberg RA, Janssen DB.

Coenzyme A ligases play an important role in metabolism by catalyzing the
activation of carboxylic acids. In this study we describe the synthesis of
aminoacyl-coenzyme As (CoAs) catalyzed by a CoA ligase from Penicillium
chrysogenum. The enzyme accepted medium-chain length fatty acids as the best
substrates, but the proteinogenic amino acids L-phenylalanine and L-tyrosine, as 
well as the non-proteinogenic amino acids D-phenylalanine, D-tyrosine and (R)-
and (S)-β-phenylalanine were also accepted. Of these amino acids, the highest
activity was found for (R)-β-phenylalanine, forming (R)-β-phenylalanyl-CoA.
Homology modeling suggested that alanine 312 is part of the active site cavity,
and mutagenesis (A312G) yielded a variant that has an enhanced catalytic
efficiency with β-phenylalanines and D-α-phenylalanine.
DOI: 10.1016/j.febslet.2011.02.018 
PMID: 21334330  [Indexed for MEDLINE]

**Comment by ValWood** [comment]
But this isn't  "tRNA aminoacylation?"

**Comment by hdrabkin** [comment]
Yes, but I cannot then merge amino acid activation into it, because there are other was to activate an aminoacid that does not involve tRNA aminoacylation. 
You can form
aa-CoA as well as aa-TRNA; 
The term amin acid activation does not specifiy which (The modification of an amino acid to an active form, for incorporation into a peptide, protein or other macromolecule.)

**Comment by ValWood** [comment]
oh right
GO:0043041    amino acid activation for nonribosomal peptide biosynthetic process

it probably isn't a particularly meaningful process grouping term? and it isn't used in annotation, so obsoletion might be better?

**Comment by hdrabkin** [comment]
unless it would be needed to house the non-tRNA dependent aa activations -(

**Comment by ValWood** [comment]
won't 
GO:0043041    amino acid activation for nonribosomal peptide biosynthetic process
do that?

**Comment by hdrabkin** [comment]
Is that a child or aminoacid activation or of aminoacyl-tRNA activation?

**Comment by hdrabkin** [comment]
That is; amino acid activation for nonribosomal peptide biosynthetic process can include aa-tRNA activation also (eg, aa-TRNA protein transfer still results in a peptide bond...

**Comment by ValWood** [comment]
These seem like common activities, that should be connected at the MF level, but not at the process level.

.... non-ribosomal  peptide synthesis is a *totally* different process to translation (genes, species, products etc, etc, etc....... so this seems a strange axis of classification in the process ontology? it's not really a biologically related "module"  other than both processes produce  a peptide bond......this seems like an over-complication in Biological  Process....

@pgaudet ?

**Comment by hdrabkin** [comment]
The whole mess started because people think that tRNA aminoacylation should have some connection to translation, so the terms aa-tRNA activation for translation was made, and then anything. 
Once you get the aa-tRNA, one can use it for translation; or other things, depending upon which aa-tRNA you have.  Yes it's complicated because aa-TRNAs have been hijacked several times. 
The problem term is the 'amino acid activation', because it turns out that there is more than one way to activate an amino acid.

**Comment by ValWood** [comment]
Hi Harold.

 This can be fixed easily.

![trna](https://user-images.githubusercontent.com/7359272/38294093-aedb748a-37e1-11e8-8236-658dd139a14b.jpg)

1. We can remove "amino acid activation" (obsolete). It isn't required. It isn't used in direct annotation.
Everything is annotated to one of the children. It isn't necessary to couple "activation" for translation and non-ribosomal peptide synthesis? The important thing is that  tRNA synthases involved in translation are coupled to translation (for enrichment and slimming).
Everything will be fine if you remove this term, it serves no useful purpose for biological process grouping.
This will remove the problem connection to amino acid metabolism.

2. Should these even be a term "tRNA aminoacylation for nonribosomal peptide biosynthetic process" this process doesn't use tRNAs does it?
(it has no annotations!)

There are probably too many terms around here that could be handled in other ways now that we can couple MF to BP easily, but this will resolve the immediate problem of parentage causing these gene products to be annotated to "amino acid metabolism"

Would that work for you? Or am I missing something?

**Comment by ValWood** [comment]
Even the def of this term is a problem
GO:0043038    amino acid activation
The modification of an amino acid to an active form, for incorporation into a peptide, protein or other macromolecule.

it seems to fit "activation for non ribosomal peptide synthesis" better than "activation for translation" (I would not call tRNA charging for peptide synthesis and *amino acid* modification?

These changes would make
GO:0043041   amino acid activation for nonribosomal peptide biosynthetic process
a bit redundant too.

You would only need
GO:0043042    amino acid adenylylation by nonribosomal peptide synthase
unless there is another way to activate amino acids for nonribosomal peptide biosynthetic process?

"activation" seems a bit "wooly"  we are forcing a connection between processes that are not really biologically related at the process level. They have more in common at the level of molecular function.

**Comment by ValWood** [comment]
PS I realise that aminoacylated tRNAs are used in other contexts, but the alternative processes aren't represented in the graph here as far as I can see?
I know I have an example of this but I can't find it....

**Comment by hdrabkin** [comment]
It appears that Cyclodipeptide synthases use an amino-acyl tRNA as a substrate. 
However,  there are two ways to synthesize a cylodipeptide, depending organisms

Below is summary from PMID: 21325056
1. Most cyclodipeptide biosynthesis pathways involve nonribosomal peptide synthetases (NRPSs) ( NRPSs are large multifunctional modular enzymes in which each module catalyzes a single step of peptide synthesis by incorporating one amino acid. In the standard NRPS module, the peptide bond-forming condensation domain (C) and the amino acid adenylation domain (A) are linked through a phosphopantetheine-tethered domain (T) to synthesize the peptidyl chain as a growing thioester. The enzyme usually ends with a thioesterase domain (TE), which releases the peptidyl chain by hydrolysis or cyclization (5). (this is more in line with  GO:0043042 
2. However, in 2002 Lautru et al. identified the albC gene in the genome of the bacterium Streptomyces noursei, and its product AlbC, which lacked similarity to the NRPSs, was able to catalyze the formation of a cyclic dipeptide (6). AlbC is the first enzyme in the biosynthetic pathway of albonoursin, a secreted antibacterial DKP bearing a cyclodipeptide structure. Recently, AlbC was shown to use aminoacylated tRNAs as substrates instead of free amino acids (7), thus allowing the enzyme to skip the amino acid activation step usually performed by the adenylation domain in NRPSs.
This is more in line with GO:0043040.

So, aa-tRNA is used by one, but not all paths to a cyclic dipeptide. 
And thus, there are TWO ways to activate an amino acid for non-ribosomal peptide synthesis, but only one way for ribosomal peptide synthesis. 

So if I obsolete GO:0043038 (amino acid activation) , the link to amino acid metabolism is gone, but then GO:0043041 has no is_a parent. (has a part_of to GO:0019184).
GO:0043042 is_a GO:0043041 as is GO:0043040
I need an is_a for GO:0043041
??

**Comment by ValWood** [comment]
bloody biology!

so what is "activation" molecularly?
"The modification of an amino acid to an active form, for incorporation into a peptide, protein or other macromolecule."   what does  the modified part mean? This might help to decide...

**Comment by deustp01** [comment]
"Activation" of a molecule means something like converting the molecule to a more chemically reactive form. Phosphate is activated when it's the terminal phosphate group of ATP. An amino acid can be activated by conjugating it with coenzyme A, as can a fatty acid. The activated form then can participate in reactions in which the original molecule (phosphate, amino acid, fatty acid) is covalently attached to something else in a reaction that would otherwise be energetically very unfavorable, (Fritz Lipmann and colleagues worked out the structures of ATP and coenzyme A as part of trying to understand how phosphorylation and fatty acylation work, and invented the term "high energy bond" to describe the linkage formed in the activation reaction that allowed the subsequent conjugation reaction to proceed.)

**Comment by hdrabkin** [comment]
yes, so aa-AMP and aa-CoA are 'activated forms of the amino acid. 
For tRNA, needed for the transfer or the aa to the terminal adenosine moeity of the tRNA. I

**Comment by ValWood** [comment]
They  aren't really "processes" are they? 

Perhaps get rid of BOTH
GO:0043038  AND GO:0043041

GO:0043042   the only thing without an is_a is
"amino acid adenylylation by nonribosomal peptide synthase"

this one *could* be is_a amino acid metabolism?
and part_of  "nonribosomal peptide biosynthetic process"

**Comment by ValWood** [comment]
Should I close this ticket so in future we don't need to read all the comments.

I suggest a new ticket to propose obsoletion of
GO:0043038  amino acid activation

because it represents a Molecular function in the biological process ontology

@hdrabkin  @pgaudet @deustp01  do you agree?

**Comment by ValWood** [comment]
I reread this ticket.

Should I proceed to obsolete these single-step processes?
GO:0043038 amino acid activation
GO:0043039    tRNA aminoacylation
GO:0043041    amino acid activation for nonribosomal peptide biosynthetic process

@pgaudet

**Comment by pgaudet** [comment]
Hi @edwong57 
I think we need to obsolete the entire branches, as they are MFs: 


GO:0043041 'tRNA aminoacylation for nonribosomal peptide biosynthetic process'
and child: 'amino acid adenylylation by nonribosomal peptide synthase'


**GO:0043039 tRNA aminoacylation**
- 'alanyl-tRNA aminoacylation'
- 'arginyl-tRNA aminoacylation'
- 'asparaginyl-tRNA aminoacylation'
- 'asparaginyl-tRNAAsn biosynthesis via transamidation'
- 'aspartyl-tRNA aminoacylation'
- 'cysteinyl-tRNA aminoacylation'
- 'cytoplasmic alanyl-tRNA aminoacylation'
- 'cytosolic valyl-tRNA aminoacylation'
- 'glutaminyl-tRNA aminoacylation'
- 'glutaminyl-tRNAGln biosynthesis via transamidation'
- 'glutamyl-tRNA aminoacylation'
- 'glycyl-tRNA aminoacylation'
- 'histidyl-tRNA aminoacylation'
- 'isoleucyl-tRNA aminoacylation'
- 'leucyl-tRNA aminoacylation'
- 'lysyl-tRNA aminoacylation'
- 'methionyl-tRNA aminoacylation'
- 'mitochondrial alanyl-tRNA aminoacylation'
- 'mitochondrial arginyl-tRNA aminoacylation'
- 'mitochondrial asparaginyl-tRNA aminoacylation'
- 'mitochondrial aspartyl-tRNA aminoacylation'
- 'mitochondrial cysteinyl-tRNA aminoacylation'
- 'mitochondrial glutaminyl-tRNA aminoacylation'
- 'mitochondrial glutamyl-tRNA aminoacylation'
- 'mitochondrial glycyl-tRNA aminoacylation'
- 'mitochondrial histidyl-tRNA aminoacylation'
- 'mitochondrial isoleucyl-tRNA aminoacylation'
- 'mitochondrial leucyl-tRNA aminoacylation'
- 'mitochondrial lysyl-tRNA aminoacylation'
- 'mitochondrial methionyl-tRNA aminoacylation'
- 'mitochondrial phenylalanyl-tRNA aminoacylation'
- 'mitochondrial prolyl-tRNA aminoacylation'
- 'mitochondrial seryl-tRNA aminoacylation'
- 'mitochondrial threonyl-tRNA aminoacylation'
- 'mitochondrial tryptophanyl-tRNA aminoacylation'
- 'mitochondrial tyrosyl-tRNA aminoacylation'
- 'mitochondrial valyl-tRNA aminoacylation'
- 'phenylalanyl-tRNA aminoacylation'
- 'prolyl-tRNA aminoacylation'
- 'seryl-tRNA aminoacylation'
- 'tRNA aminoacylation for mitochondrial protein translation'
- 'tRNA aminoacylation for nonribosomal peptide biosynthetic process'
- 'tRNA aminoacylation for protein translation'
- 'threonyl-tRNA aminoacylation'
- 'tryptophanyl-tRNA aminoacylation'
- 'tyrosyl-tRNA aminoacylation'
- 'valyl-tRNA aminoacylation'

**GO:0043038 amino acid activation** & child
'alanyl-tRNA aminoacylation'
'amino acid activation for nonribosomal peptide biosynthetic process'
'amino acid adenylylation by nonribosomal peptide synthase'
'arginyl-tRNA aminoacylation'
'asparaginyl-tRNA aminoacylation'
'asparaginyl-tRNAAsn biosynthesis via transamidation'
'aspartyl-tRNA aminoacylation'
'cysteinyl-tRNA aminoacylation'
'cytoplasmic alanyl-tRNA aminoacylation'
'cytosolic valyl-tRNA aminoacylation'
'glutaminyl-tRNA aminoacylation'
'glutaminyl-tRNAGln biosynthesis via transamidation'
'glutamyl-tRNA aminoacylation'
'glycyl-tRNA aminoacylation'
'histidyl-tRNA aminoacylation'
'isoleucyl-tRNA aminoacylation'
'leucyl-tRNA aminoacylation'
'lysyl-tRNA aminoacylation'
'methionyl-tRNA aminoacylation'
'mitochondrial alanyl-tRNA aminoacylation'
'mitochondrial arginyl-tRNA aminoacylation'
'mitochondrial asparaginyl-tRNA aminoacylation'
'mitochondrial aspartyl-tRNA aminoacylation'
'mitochondrial cysteinyl-tRNA aminoacylation'
'mitochondrial glutaminyl-tRNA aminoacylation'
'mitochondrial glutamyl-tRNA aminoacylation'
'mitochondrial glycyl-tRNA aminoacylation'
'mitochondrial histidyl-tRNA aminoacylation'
'mitochondrial isoleucyl-tRNA aminoacylation'
'mitochondrial leucyl-tRNA aminoacylation'
'mitochondrial lysyl-tRNA aminoacylation'
'mitochondrial methionyl-tRNA aminoacylation'
'mitochondrial phenylalanyl-tRNA aminoacylation'
'mitochondrial prolyl-tRNA aminoacylation'
'mitochondrial seryl-tRNA aminoacylation'
'mitochondrial threonyl-tRNA aminoacylation'
'mitochondrial tryptophanyl-tRNA aminoacylation'
'mitochondrial tyrosyl-tRNA aminoacylation'
'mitochondrial valyl-tRNA aminoacylation'
'phenylalanyl-tRNA aminoacylation'
'prolyl-tRNA aminoacylation'
'seryl-tRNA aminoacylation'
'tRNA aminoacylation for mitochondrial protein translation'
'tRNA aminoacylation for nonribosomal peptide biosynthetic process'
'tRNA aminoacylation for protein translation'
'tRNA aminoacylation'
'threonyl-tRNA aminoacylation'
'tryptophanyl-tRNA aminoacylation'
'tyrosyl-tRNA aminoacylation'
'valyl-tRNA aminoacylation'


If you can query annotations, and let me know when we're OK to obsolete, I can do the obsoletion in bulk. 

Thanks, Pascale

**Comment by ValWood** [comment]
These terms have "amino acid metabolism" and "tRNA metabolism"  and "cytoplasmic/mitochondrial translation" in their parentage.

Which do you think is correct? I was thinking "translation" but this isn't right because aminoacylationis pres translation iniitation.
The metabolism terms are a bit strange here too (the tRNA isn't metabolised)

What do you think?

**Comment by ValWood** [comment]
'glutaminyl-tRNAGln biosynthesis via transamidation' looks valid

A tRNA aminoacylation process in which glutaminyl-tRNAGln is formed by a tRNA-dependent two-step pathway. In the first step a non-discriminating glutamyl-tRNAGlx synthetase generates the misacylated L-glutamyl-tRNAGln species, and in the second step it is amidated to the correctly charged L-glutaminyl-tRNAGln by a glutamyl-tRNAGln amidotransferase.

**Comment by edwong57** [comment]
> Hi [@edwong57](https://github.com/edwong57) I think we need to obsolete the entire branches, as they are MFs:
> 
> * GO:0043041 'tRNA aminoacylation for nonribosomal peptide biosynthetic process'
> * and child: 'amino acid adenylylation by nonribosomal peptide synthase'
> * **GO:0043039 tRNA aminoacylation**
>   and children
> * 'alanyl-tRNA aminoacylation'
> * 'arginyl-tRNA aminoacylation'
> * 'asparaginyl-tRNA aminoacylation'
> * 'asparaginyl-tRNAAsn biosynthesis via transamidation'
> * 'aspartyl-tRNA aminoacylation'
> * 'cysteinyl-tRNA aminoacylation'
> * 'cytoplasmic alanyl-tRNA aminoacylation'
> * 'cytosolic valyl-tRNA aminoacylation'
> * 'glutaminyl-tRNA aminoacylation'
> * 'glutaminyl-tRNAGln biosynthesis via transamidation'
> * 'glutamyl-tRNA aminoacylation'
> * 'glycyl-tRNA aminoacylation'
> * 'histidyl-tRNA aminoacylation'
> * 'isoleucyl-tRNA aminoacylation'
> * 'leucyl-tRNA aminoacylation'
> * 'lysyl-tRNA aminoacylation'
> * 'methionyl-tRNA aminoacylation'
> * 'mitochondrial alanyl-tRNA aminoacylation'
> * 'mitochondrial arginyl-tRNA aminoacylation'
> * 'mitochondrial asparaginyl-tRNA aminoacylation'
> * 'mitochondrial aspartyl-tRNA aminoacylation'
> * 'mitochondrial cysteinyl-tRNA aminoacylation'
> * 'mitochondrial glutaminyl-tRNA aminoacylation'
> * 'mitochondrial glutamyl-tRNA aminoacylation'
> * 'mitochondrial glycyl-tRNA aminoacylation'
> * 'mitochondrial histidyl-tRNA aminoacylation'
> * 'mitochondrial isoleucyl-tRNA aminoacylation'
> * 'mitochondrial leucyl-tRNA aminoacylation'
> * 'mitochondrial lysyl-tRNA aminoacylation'
> * 'mitochondrial methionyl-tRNA aminoacylation'
> * 'mitochondrial phenylalanyl-tRNA aminoacylation'
> * 'mitochondrial prolyl-tRNA aminoacylation'
> * 'mitochondrial seryl-tRNA aminoacylation'
> * 'mitochondrial threonyl-tRNA aminoacylation'
> * 'mitochondrial tryptophanyl-tRNA aminoacylation'
> * 'mitochondrial tyrosyl-tRNA aminoacylation'
> * 'mitochondrial valyl-tRNA aminoacylation'
> * 'phenylalanyl-tRNA aminoacylation'
> * 'prolyl-tRNA aminoacylation'
> * 'seryl-tRNA aminoacylation'
> * 'tRNA aminoacylation for mitochondrial protein translation'
> * 'tRNA aminoacylation for nonribosomal peptide biosynthetic process'
> * 'tRNA aminoacylation for protein translation'
> * 'threonyl-tRNA aminoacylation'
> * 'tryptophanyl-tRNA aminoacylation'
> * 'tyrosyl-tRNA aminoacylation'
> * 'valyl-tRNA aminoacylation'
> * **GO:0043038 amino acid activation** & children
> * 'alanyl-tRNA aminoacylation'
> * 'amino acid activation for nonribosomal peptide biosynthetic process'
> * 'amino acid adenylylation by nonribosomal peptide synthase'
> * 'arginyl-tRNA aminoacylation'
> * 'asparaginyl-tRNA aminoacylation'
> * 'asparaginyl-tRNAAsn biosynthesis via transamidation'
> * 'aspartyl-tRNA aminoacylation'
> * 'cysteinyl-tRNA aminoacylation'
> * 'cytoplasmic alanyl-tRNA aminoacylation'
> * 'cytosolic valyl-tRNA aminoacylation'
> * 'glutaminyl-tRNA aminoacylation'
> * 'glutaminyl-tRNAGln biosynthesis via transamidation'
> * 'glutamyl-tRNA aminoacylation'
> * 'glycyl-tRNA aminoacylation'
> * 'histidyl-tRNA aminoacylation'
> * 'isoleucyl-tRNA aminoacylation'
> * 'leucyl-tRNA aminoacylation'
> * 'lysyl-tRNA aminoacylation'
> * 'methionyl-tRNA aminoacylation'
> * 'mitochondrial alanyl-tRNA aminoacylation'
> * 'mitochondrial arginyl-tRNA aminoacylation'
> * 'mitochondrial asparaginyl-tRNA aminoacylation'
> * 'mitochondrial aspartyl-tRNA aminoacylation'
> * 'mitochondrial cysteinyl-tRNA aminoacylation'
> * 'mitochondrial glutaminyl-tRNA aminoacylation'
> * 'mitochondrial glutamyl-tRNA aminoacylation'
> * 'mitochondrial glycyl-tRNA aminoacylation'
> * 'mitochondrial histidyl-tRNA aminoacylation'
> * 'mitochondrial isoleucyl-tRNA aminoacylation'
> * 'mitochondrial leucyl-tRNA aminoacylation'
> * 'mitochondrial lysyl-tRNA aminoacylation'
> * 'mitochondrial methionyl-tRNA aminoacylation'
> * 'mitochondrial phenylalanyl-tRNA aminoacylation'
> * 'mitochondrial prolyl-tRNA aminoacylation'
> * 'mitochondrial seryl-tRNA aminoacylation'
> * 'mitochondrial threonyl-tRNA aminoacylation'
> * 'mitochondrial tryptophanyl-tRNA aminoacylation'
> * 'mitochondrial tyrosyl-tRNA aminoacylation'
> * 'mitochondrial valyl-tRNA aminoacylation'
> * 'phenylalanyl-tRNA aminoacylation'
> * 'prolyl-tRNA aminoacylation'
> * 'seryl-tRNA aminoacylation'
> * 'tRNA aminoacylation for mitochondrial protein translation'
> * 'tRNA aminoacylation for nonribosomal peptide biosynthetic process'
> * 'tRNA aminoacylation for protein translation'
> * 'tRNA aminoacylation'
> * 'threonyl-tRNA aminoacylation'
> * 'tryptophanyl-tRNA aminoacylation'
> * 'tyrosyl-tRNA aminoacylation'
> * 'valyl-tRNA aminoacylation'
> 
> If you can query annotations, and let me know when we're OK to obsolete, I can do the obsoletion in bulk.
> 
> Thanks, Pascale

@pgaudet, just want to make sure that I understand correctly. By query, do you mean run the script to find all the annotations for these terms, and make an annotation review ticket?

**Comment by pgaudet** [comment]
> By query, do you mean run the script to find all the annotations for these terms, and make an annotation review ticket?

Yes

**Comment by ValWood** [comment]
What term will tRNA aminoacylation be reannotated to?

I don't like the current parents ;)
 "tRNA charging" should probably be part_of a protein synthesis term....
This will also help with other parts of protein synthesis that are not part of translation like ribosome recycling.
At the moment we had some terms directly under translation (which was incorrect)

<img width="554" alt="Image" src="https://github.com/user-attachments/assets/2d9ada6f-d704-4b87-a015-9a5bf636d656" />

we also had parents to amino acid metabolism (incorrect?) and tRNA metabolism (also incorrect?)

**Comment by edwong57** [comment]
Annotations to these terms listed are on this spreadsheet - https://docs.google.com/spreadsheets/d/1oop8zlFAJPO9Jpn_nsz789es5l06RoRSEFAHk9sswEU/edit?usp=sharing

Query on this ticket - https://github.com/geneontology/annotation-query/issues/24

Included GO:0070681	glutaminyl-tRNAGln biosynthesis via transamidation which looks like has 'NOT THIS TERM' on one of the comments.

**Comment by ValWood** [comment]
@deustp01  messaged:

First, if you treat the tRNA as a chemical entity rather than as an encoded macromolecule, ChEBI solves the modified entity problem. E.g. tRNAGlu (CHEBI:29175) is transformed into Glu-tRNAGlu CHEBI:29157. I know there is an RNA ontology out there someplace, but for now perhaps treating tRNA as an ordinary chemical maximizes the information that can be preserved within a GO-CAM model. Rhea also has a way to annotate this reaction, [https://www.rhea-db.org/rhea/23541](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.rhea-db.org%2Frhea%2F23541&data=05%7C02%7Cvw253%40universityofcambridgecloud.onmicrosoft.com%7Cfc3829c39e7b4bc13ada08dd68bce737%7C49a50445bdfa4b79ade3547b4f3986e9%7C1%7C0%7C638781882728699640%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=wa1nt4%2FxJvjLDRvsDuiZqHswdEDItfIc4ee1UZOrDQE%3D&reserved=0), using its own strategy for handling the tRNA chemical entity before and after glutamate addition.

I have now modelled the inputs using PomBase entities, but because we don't have a public RNA modifications ontology (I also know there is one, but I don't think it is publicly useable), I have now modelled the outputs using the CHEBI charged forms (good tip)

**Comment by ValWood** [comment]
@deustp01 messaged:

Second, the Stryer and Devlin biochemistry texts both treat aminoacylation of tRNAs as a step in the process of translation / protein synthesis. Unless aminoacylated tRNAs have a role in some other part of biology entirely, this looks like a good community-consensus place to locate tRNA aminoacylation in GO. At the same time, I think I see a true path violation with this arrangement, because now GO:0006412 translation is_a protein metabolic process, and GO:0043039 tRNA aminoacylation is_a RNA metabolic process. Or can we evade that by saying translation has_part aminoacylation (new relationship) while preserving the existing is_a relationships?

**Comment by ValWood** [comment]
> Second, the Stryer and Devlin biochemistry texts both treat aminoacylation of tRNAs as a step in the process of translation / protein synthesis. 

I agree it should be part of protein synthesis (which does not exist ATM in GO.
I'm not sure it is formally part of translation.

translation
The cellular metabolic process in which a protein is formed, using the sequence of a mature mRNA or circRNA molecule to specify the sequence of amino acids in a polypeptide chain. Translation is mediated by the ribosome, and begins with the formation of a ternary complex between aminoacylated initiator methionine tRNA, GTP, and initiation factor 2, which subsequently associates with the small subunit of the ribosome and an mRNA or circRNA. Translation ends with the release of a polypeptide chain from the ribosome.

aminoacylation (except for the f-met), would need to be part of elongation.

Elongation is defined
The successive addition of amino acid residues to a nascent polypeptide chain during protein biosynthesis.

it seems to be pert of protein synthesis, but causally upstream of elongation?

**Comment by ValWood** [comment]
ChatGPT Q Is amino acylation part of translation?

A:
Aminoacylation (also known as tRNA charging) is a prerequisite for translation but is not considered part of the translation process itself.

Explanation:
Aminoacylation refers to the process by which an aminoacyl-tRNA synthetase enzyme attaches an amino acid to its corresponding tRNA. This ensures that the tRNA is properly charged and ready to participate in protein synthesis.

Translation officially begins with initiation, where the ribosome assembles around the mRNA and the first charged tRNA binds to the start codon.

So, while aminoacylation is essential for translation, it occurs before translation begins and is not technically part of it.

**Comment by pgaudet** [comment]
Decision on the annotation call: 

(1) 
- NTR protein synthesis 
- - part_of translation
- - - part_of translation initiation
- - - part_of translation elongation
- - -  part_of translation termination
- part_of 'tRNA aminoacylation' >> rename 'tRNA charging'? add synonyms: aminoacyl tRNA synthesis, amino acid activation

(2) 
same for cytosolic/cytoplasmic, mitochondrion and plastid 

(3) Fix parents: [translation](http://purl.obolibrary.org/obo/GO_0006412): 
- remove asserted superclasses [protein metabolic process](http://purl.obolibrary.org/obo/GO_0019538) and [macromolecule biosynthetic process](http://purl.obolibrary.org/obo/GO_0009059)

**Comment by raymond91125** [comment]
What should the relationship be between GO:0010467 gene expression and protein biosynthetic process?

+[Term]
+id: GO:0160307
+name: protein biosynthetic process
+namespace: biological_process
+def: "The chemical reactions and pathways resulting in the formation of protein." [WIKIPEDIA:Protein_biosynthesis]
+synonym: "protein biosynthesis" EXACT []
+synonym: "protein synthesis" EXACT []
+intersection_of: GO:0009059 ! macromolecule biosynthetic process
+intersection_of: has_primary_output PR:000000001 ! protein
+property_value: term_tracker_item "https://github.com/geneontology/go-ontology/issues/15375" xsd:anyURI
+created_by: rynl
+creation_date: 2025-09-29T23:16:32Z

**Comment by pgaudet** [comment]
> What should the relationship be between GO:0010467 gene expression and protein biosynthetic process?

- protein biosynthetic process should be **part_of** gene expression 
Same as GO:0006412 translation, GO:0006351 DNA-templated transcription, GO:0006396 RNA processing, and GO:0051604 protein maturation.

Thanks, Pascale

**Comment by raymond91125** [comment]
The problem is then translation is equivalent to protein biosynthetic process. Because translation is a widely used slim term, I feel reluctant to mess with it.

**Comment by ValWood** [comment]
Hi @raymond91125 

protein synthesis will be

NTR protein synthesis
-part_of translation
-part_of 'tRNA aminoacylation' >> rename 'tRNA charging'

it will have 2 parts , translation and tRNA charging (renaming tRNA aminoacylation so it is less function-y)
All of the other terms related to  'tRNA aminoacylation' and "amino acid activation" will be obsoleted.

Trnaslation is only one part of protein synthesis, the other part is tRNA charging

**Comment by ValWood** [comment]
So tRNA charging will also be part of gene expression...this makes sense to me

**Comment by pgaudet** [comment]
But there is a problem WRT logical definitions, see 
https://github.com/geneontology/go-ontology/pull/30876

@raymond91125 you can probably add a MF in the logical definition, [peptidyltransferase activity](http://purl.obolibrary.org/obo/GO_0000048) (I think that's the correct reaction?)

**Comment by raymond91125** [comment]
It seems protein biosynthetic process would also be has_part peptidyltransferase activity. Perhaps the distinguishing feature is that protein biosynthetic process has_part tRNA aminoacylation. tRNA aminoacylation does include tRNA aminoacylation for nonribosomal peptide biosynthetic process thus it is not strictly part of protein biosynthetic process.

**Comment by raymond91125** [comment]
Merged. Please reopen if there are any unresolved.

**Comment by ValWood** [comment]
The other part of this ticket  was to:
 
- obsolete the BP terms  for individual  tRNA aminoacylation 

GO:0006420    arginyl-tRNA aminoacylation
GO:0006432    phenylalanyl-tRNA aminoacylation
GO:0006433    prolyl-tRNA aminoacylation
GO:0006422    aspartyl-tRNA aminoacylation
GO:0006430    lysyl-tRNA aminoacylation
GO:0006421    asparaginyl-tRNA aminoacylation
GO:0006431    methionyl-tRNA aminoacylation
GO:0006424    glutamyl-tRNA aminoacylation
GO:0006436    tryptophanyl-tRNA aminoacylation
GO:0006423    cysteinyl-tRNA aminoacylation
etc

- obsolete unnecessary superclass GO:0043038   amino acid activation

- make tRNA aminoacylation  a subclass "protein biosynthetic process"
 
- rename tRNA aminoacylation to "tRNA charging"

**Comment by raymond91125** [comment]
Previously, there was a list of terms for obsoletion. GO:0043039 tRNA aminoacylation is in that list. Does that list need to be revised?
https://github.com/geneontology/go-ontology/issues/15375#issuecomment-2669074122
@edwong57 found 290 annotations that need to be dealt with based on that list. Are there specific terms to move them to?

**Comment by ValWood** [comment]
If I remember correctly, all annotations would move to 
tRNA aminoacylation  which will be renamed to  "tRNA charging"
so that we reduce the process branch to a single term.

Presumably, this can be automated with a direct replaced_by  for most databases?

@pgaudet  was this your recollection?

I think originally I thought we might keep the parent "GO:0043038   
amino acid activation" so I can't remember exactly why we switched from
GO:0043038   amino acid activation" to GO:0043039  tRNA aminoacylation
(I have no preference)

<img width="878" height="873" alt="Image" src="https://github.com/user-attachments/assets/7642df15-130f-47af-8898-ee65880855c6" />

**Comment by ValWood** [comment]
...but whichever term is retained needs:
i) GO:0160307  protein biosynthetic process as a parent
ii)  amino acid biosynthetic process, AND translation should not be parents (because aminoacylation is not part of either of these processes)

**Comment by raymond91125** [comment]
@pgaudet Could you confirm that we've decided to keep GO:0043039 tRNA aminoacylation and replace all its children term with it?

GO:0043038 amino acid activation has two child terms which make it NOT an unnecessary grouping term. Am I missing something?
.....GO:0043041 amino acid activation for nonribosomal peptide biosynthetic process
.....GO:0043039 tRNA aminoacylation

GO distinguishes GO:0043043 peptide biosynthetic process from GO:0160307 protein biosynthetic process. e.g. GO:0002936 bradykinin biosynthetic process and GO:0030651 peptide antibiotic biosynthetic process are children of GO:0043043 peptide biosynthetic process that (at least some times) use ribosomes.

**Comment by ValWood** [comment]
see also
https://github.com/geneontology/go-ontology/issues/30872

**Comment by raymond91125** [comment]
@dragon-ai-agent please go over all comments in this issue to figure out what else needs done. Just provide the summary, not a pull request.

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23872058360)

*— @dragon-ai-agent*

**Comment by dragon-ai-agent** [comment]
## Summary of remaining work on this issue

### What has been done

- **GO:0160307 protein biosynthetic process** was created (PR #30876, merged 2025-10-01) with `part_of gene expression` and `has_part tRNA aminoacylation` in its logical definition.
- **GO:0006412 translation** now has `part_of GO:0160307 protein biosynthetic process`.
- **Annotation query** was run by @edwong57 — 290 annotations identified across the terms slated for obsoletion (spreadsheet linked in comment, annotation-query issue geneontology/annotation-query#24 still open).

### What still needs to be done

The following items have been agreed upon in the discussion but not yet implemented:

#### 1. Bulk obsoletion of individual tRNA aminoacylation BP terms (~45 terms)

All children of GO:0043039 should be obsoleted and replaced by GO:0043039 itself. This includes:
- **20 amino acid-specific terms** (GO:0006419–GO:0006438): alanyl-, arginyl-, asparaginyl-, ..., valyl-tRNA aminoacylation
- **17 mitochondrial-specific terms** (GO:0070143–GO:0070159): mitochondrial alanyl-, arginyl-, ..., valyl-tRNA aminoacylation
- **GO:0006418** tRNA aminoacylation for protein translation
- **GO:0070127** tRNA aminoacylation for mitochondrial protein translation
- **GO:0043040** tRNA aminoacylation for nonribosomal peptide biosynthetic process
- **GO:0070680** asparaginyl-tRNAAsn biosynthesis via transamidation
- **GO:0070681** glutaminyl-tRNAGln biosynthesis via transamidation (though @ValWood flagged this one as possibly valid — it describes a genuine two-step process)

All of these should get `replaced_by: GO:0043039` to facilitate automated annotation migration.

#### 2. Obsoletion of grouping/parent terms

- **GO:0043038** amino acid activation — obsolete, replace with GO:0043039
- **GO:0043041** amino acid activation for nonribosomal peptide biosynthetic process — obsolete (also tracked in issue #30872)
- **GO:0043042** amino acid adenylylation by nonribosomal peptide synthase — obsolete (child of GO:0043041)

#### 3. Rename GO:0043039

- Rename `tRNA aminoacylation` to `tRNA charging` (per annotation call decision, comment #50)
- Add synonyms: "aminoacyl tRNA synthesis", "amino acid activation", "tRNA aminoacylation"

#### 4. Fix parentage of GO:0043039

Per @ValWood (comment #62), the retained term needs:
- **Add**: `part_of GO:0160307 protein biosynthetic process` (or possibly `is_a`)
- **Remove**: `is_a GO:0006399 tRNA metabolic process` (currently asserted — tRNA is not metabolized)
- **Remove**: `is_a GO:0043038 amino acid activation` (will be obsoleted)
- **Confirm**: translation should NOT be a parent (aminoacylation precedes translation)

#### 5. Unresolved questions awaiting @pgaudet confirmation

- **Comment #63 from @raymond91125**: Confirmation requested that GO:0043039 (not GO:0043038) is the term to keep, and that all children should be replaced by it. No response yet.
- **Comment #63**: Whether GO:0043038 should be obsoleted given it has two children (GO:0043041 and GO:0043039) — is it truly unnecessary?
- **GO:0070681 glutaminyl-tRNAGln biosynthesis via transamidation** — should this be kept as a valid multi-step process term, or obsoleted with the rest?
- **Logical definition of GO:0160307 protein biosynthetic process** (comment #56–57): @pgaudet suggested adding peptidyltransferase activity to the logical definition; @raymond91125 noted this might also apply to protein biosynthetic process, and that the distinguishing feature might be `has_part tRNA aminoacylation`. The logical definition may need revisiting.
- **Cytoplasmic/plastid parallels** (comment #50, item 2): The annotation call decided that cytosolic/cytoplasmic, mitochondrial, and plastid-specific forms should follow the same pattern. The mitochondrial terms exist but cytoplasmic/plastid-specific protein biosynthetic process terms may need to be created or adjusted.

#### 6. Annotation migration

- 290 annotations need to be migrated (per @edwong57's spreadsheet)
- Most should be automatable via `replaced_by` directives
- The annotation-query ticket (geneontology/annotation-query#24) is still open

### Suggested next step

Get @pgaudet's response to @raymond91125's question in comment #63 before proceeding with the bulk obsoletion.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-6`
- Agent harness: claude-code
- Triggered by: @raymond91125
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23872058360)

**Comment by pgaudet** [comment]
I dont understand how https://github.com/geneontology/go-ontology/issues/63 is related ?

**Comment by raymond91125** [comment]
@dragon-ai-agent please correct references to comments. The URL links are incorrect and the numbering system is not comprehensible. Use page section links and first sentences to reference comments.

**Comment by dragon-ai-agent** [comment]
🤖 Working on it...

Follow along: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23914382771)

*— @dragon-ai-agent*

**Comment by dragon-ai-agent** [comment]
## Corrected summary of remaining work on this issue

(Reposting my [earlier summary](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-4173125702) with fixed comment references. Apologies for the confusion — in particular, "comment #63" was not a reference to issue #63.)

### What has been done

- **GO:0160307 protein biosynthetic process** was created (PR #30876, merged 2025-10-01) with `part_of gene expression` and `has_part tRNA aminoacylation` in its logical definition.
- **GO:0006412 translation** now has `part_of GO:0160307 protein biosynthetic process`.
- **Annotation query** was run by @edwong57 — 290 annotations identified across the terms slated for obsoletion (spreadsheet linked in [their comment](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-2744689419), "Annotations to these terms listed are on this spreadsheet..."); annotation-query issue geneontology/annotation-query#24 still open.

### What still needs to be done

#### 1. Bulk obsoletion of individual tRNA aminoacylation BP terms (~45 terms)

All children of GO:0043039 should be obsoleted and replaced by GO:0043039 itself. This includes:
- **20 amino acid-specific terms** (GO:0006419–GO:0006438): alanyl-, arginyl-, asparaginyl-, ..., valyl-tRNA aminoacylation
- **17 mitochondrial-specific terms** (GO:0070143–GO:0070159): mitochondrial alanyl-, arginyl-, ..., valyl-tRNA aminoacylation
- **GO:0006418** tRNA aminoacylation for protein translation
- **GO:0070127** tRNA aminoacylation for mitochondrial protein translation
- **GO:0043040** tRNA aminoacylation for nonribosomal peptide biosynthetic process
- **GO:0070680** asparaginyl-tRNAAsn biosynthesis via transamidation
- **GO:0070681** glutaminyl-tRNAGln biosynthesis via transamidation (though @ValWood [flagged this as possibly valid](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-2669422582): "'glutaminyl-tRNAGln biosynthesis via transamidation' looks valid — A tRNA aminoacylation process in which glutaminyl-tRNAGln is formed by a tRNA-dependent two-step pathway...")

All of these should get `replaced_by: GO:0043039` to facilitate automated annotation migration.

#### 2. Obsoletion of grouping/parent terms

- **GO:0043038** amino acid activation — obsolete, replace with GO:0043039
- **GO:0043041** amino acid activation for nonribosomal peptide biosynthetic process — obsolete (also tracked in #30872)
- **GO:0043042** amino acid adenylylation by nonribosomal peptide synthase — obsolete (child of GO:0043041)

#### 3. Rename GO:0043039

- Rename `tRNA aminoacylation` to `tRNA charging` (per @pgaudet's [annotation call decision](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3347997731): "Decision on the annotation call... part_of 'tRNA aminoacylation' >> rename 'tRNA charging'? add synonyms...")
- Add synonyms: "aminoacyl tRNA synthesis", "amino acid activation", "tRNA aminoacylation"

#### 4. Fix parentage of GO:0043039

Per @ValWood ([comment](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3714855438): "...but whichever term is retained needs: i) GO:0160307 protein biosynthetic process as a parent..."), the retained term needs:
- **Add**: `part_of GO:0160307 protein biosynthetic process`
- **Remove**: `is_a GO:0006399 tRNA metabolic process` (currently asserted — tRNA is not metabolized)
- **Remove**: `is_a GO:0043038 amino acid activation` (will be obsoleted)
- **Confirm**: translation should NOT be a parent (aminoacylation precedes translation)

#### 5. Unresolved questions awaiting @pgaudet confirmation

- @raymond91125 [asked](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3717016402) ("Could you confirm that we've decided to keep GO:0043039 tRNA aminoacylation and replace all its children term with it?"): Confirmation requested that GO:0043039 (not GO:0043038) is the term to keep, and that all children should be replaced by it.
- Same comment: Whether GO:0043038 should be obsoleted given it has two children (GO:0043041 and GO:0043039).
- **GO:0070681 glutaminyl-tRNAGln biosynthesis via transamidation** — should this be kept as a valid multi-step process term, or obsoleted with the rest?
- @pgaudet [raised a concern about logical definitions](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3355759974) ("But there is a problem WRT logical definitions, see PR #30876... you can probably add a MF in the logical definition, peptidyltransferase activity"); @raymond91125 [responded](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3357475765) ("It seems protein biosynthetic process would also be has_part peptidyltransferase activity..."). The logical definition of GO:0160307 may need revisiting.
- @pgaudet's annotation call decision ([comment](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3347997731), item 2: "same for cytosolic/cytoplasmic, mitochondrion and plastid"): Cytoplasmic/plastid parallels — the mitochondrial terms exist but cytoplasmic/plastid-specific protein biosynthetic process terms may need to be created or adjusted.

#### 6. Annotation migration

- 290 annotations need to be migrated (per @edwong57's [spreadsheet](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-2744689419))
- Most should be automatable via `replaced_by` directives
- The annotation-query ticket (geneontology/annotation-query#24) is still open

### Suggested next step

Get @pgaudet's response to @raymond91125's [confirmation request](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-3717016402) before proceeding with the bulk obsoletion.

---
🤖 **Generated by @dragon-ai-agent**
- Model: `claude-opus-4-6`
- Agent harness: claude-code
- Triggered by: @raymond91125
- Run: [View workflow run](https://github.com/geneontology/go-ontology/actions/runs/23914382771)

**Comment by raymond91125** [comment]
@pgaudet Should I proceed with this mass obsoletion?

**Comment by ValWood** [comment]
What's the status here?

We  were going to keep one grouping term "tRNA aminoacylation" (plus  we would require mitochondrial and cytosolic variants)   and therse terms should be part of "protein biosynthetic process" aka protein synthesis (the new grouping term)

Although, we don't need a grouping term for translation, grouping these processes by the protein synthesis term would be sufficient.

A simpler organization could be:

protein biosynthetic process
—mitochondrial protein biosynthetic process
—— mitochondrial translation
—— mitochondrial tRNA aminoacylation
—cytoplasmic (cytosolic) protein biosynthetic process
—  cytoplasmic (cytosolic) translation
—  cytosolic tTNA aminiacylation

OR  We could annotate tRNA ligase to  the respective biosynthesis terms and drop  the tRNA aminoacylation grouping terms BUT this would make  the translation tems single inheritance, which could be confusing.



CC @hattrill

**Comment by hattrill** [comment]
@ValWood Aren't you on holiday? 
Your simple organisation works for me. What would be the starts with and ends with definition of "protein biosynthetic process" for this?

**Comment by ValWood** [comment]
starts with  is tricky, it includes  amino acid charging and translation iniitation but neither "comes first' they occur in parallel....

**Comment by raymond91125** [comment]
[Written by Claude]

Opened #32537, which fixes the one part of this issue that has been settled since 2018 and needs no further confirmation. The rest of the ticket is untouched and stays open.

## What changed

`tRNA aminoacylation` is no longer a kind of `amino acid metabolic process`. That link ran through `amino acid activation`, and removing it drops tRNA aminoacylation and all ~48 of its descendants — every amino-acid-specific and mitochondrial aminoacylation term — out from under amino acid metabolism. This is exactly the action @ValWood described and @deustp01 and @hdrabkin agreed to: charging a tRNA attaches the amino acid to something, it doesn't metabolize it.

Three terms were touched:

- **`GO:0043039` tRNA aminoacylation** keeps its place under `tRNA metabolic process`, and keeps "amino acid activation" as a synonym so anyone searching the old wording still finds it.
- **`GO:0043038` amino acid activation** is obsoleted. It had no annotations at all — I re-checked against the live annotation database, still zero, so nothing had to be moved. It was only ever a grouping term over two processes with nothing in common, and once tRNA aminoacylation moved out there was nothing coherent left to define. It points at `GO:0043039` and `GO:0043041` as things to consider instead.
- **`GO:0043041` amino acid activation for nonribosomal peptide biosynthetic process** moves directly under `amino acid metabolic process`, keeping its 267 annotations and its `part_of nonribosomal peptide biosynthetic process`.

That last one is a small departure from the letter of the plan, and worth flagging. The stated action was to cut the link between `amino acid activation` and `amino acid metabolic process`. Doing it at that edge would also have pulled the nonribosomal branch out of amino acid metabolism — and that branch genuinely belongs there. Its child adenylylates the amino acid at its carboxylate group, which is real chemical modification of the amino acid. That is precisely the tRNA-independent activation @hdrabkin dug up (the aminoacyl-CoA paper, PMID:21334330). So the cut was made one level lower: tRNA aminoacylation leaves, nonribosomal activation stays.

## One thing to be aware of

Annotations to terms like `glycyl-tRNA aminoacylation` will stop rolling up to `amino acid metabolic process`. That is the point of the change, but it is a visible difference for anyone consuming inferred annotations or using a slim built on `amino acid metabolic process`.

## Still open

Everything else on this ticket: the bulk obsoletion of the ~45 specific aminoacylation terms, the rename to "tRNA charging", the `protein biosynthetic process` re-organisation and the cytosolic/mitochondrial/plastid variants @ValWood sketched, and the 290-annotation migration. Those are still waiting on @pgaudet's answer to @raymond91125's question about which term to keep. The idea here was just to get the uncontested correction in rather than have it wait on that.

@ValWood @hdrabkin @pgaudet — if you would rather this waited and went in as one piece with the larger restructuring, say so and it can be held.

**Comment by raymond91125** [comment]
[Written by Claude]

#32537 is merged. Correcting my earlier comment on one point, and listing what is still open.

## Correction: four terms changed, not three

My comment above described three term changes. Review caught a gap and the merged change touches **four**. The addition:

**`GO:0043040` tRNA aminoacylation for nonribosomal peptide biosynthetic process** — this had two parents, `tRNA aminoacylation` *and* `amino acid activation for nonribosomal peptide biosynthetic process`. Moving the latter under `amino acid metabolic process` would have left this one tRNA charging term still sitting under amino acid metabolism — the exact thing the change was meant to stop. Its second parent is now `part_of nonribosomal peptide biosynthetic process` instead, so it keeps the nonribosomal link without the metabolic inheritance. No annotations were affected.

Side effect worth knowing: `GO:0043041` now has only `GO:0043042 amino acid adenylylation by nonribosomal peptide synthase` beneath it, which makes it a coherent grouping — everything under it genuinely modifies the amino acid.

Verified by closure over `is_a` and `part_of`: the `tRNA aminoacylation` branch is 46 terms and **none** of them is under `amino acid metabolic process` any more.

## Now true in the ontology

- `tRNA aminoacylation` sits under `tRNA metabolic process`, and nothing in its branch is amino acid metabolism
- `amino acid activation` is obsolete, pointing at `tRNA aminoacylation` and `amino acid activation for nonribosomal peptide biosynthetic process`
- The nonribosomal activation branch stays under amino acid metabolism, where its chemistry belongs

Announcement went out as geneontology/go-announcements#1184.

## Still open on this ticket

Unchanged by the above:

1. Bulk obsoletion of the ~45 specific tRNA aminoacylation terms
2. Rename `tRNA aminoacylation` to `tRNA charging` plus synonyms
3. Removing `is_a tRNA metabolic process` and adding `part_of GO:0160307 protein biosynthetic process`
4. The cytosolic / mitochondrial / plastid organisation @ValWood sketched
5. Migration of the 290 annotations (geneontology/annotation-query#24)
6. The MF_in_BP / GO-CAM work

Items 1-3 still need @pgaudet's answer to @raymond91125's question about which term to keep.

## One open question from review

@pgaudet @ValWood — the plan for this ticket specified `replaced_by: GO:0043039` on the obsoleted term. It went in with two `consider:` pointers instead, because the two children are genuinely different biology and a single `replaced_by` would push any nonribosomal usage onto a tRNA charging term. With zero annotations it makes little practical difference, but `replaced_by` is what drives automatic repair if a stale annotation is sitting in a MOD's GAF or an older GO-CAM. Easy to switch if you would rather follow the original plan.

**Comment by raymond91125** [comment]
[Written by Claude]

@pgaudet @ValWood @hattrill — a concrete proposal for the rest of this ticket, aiming for the fewest moving parts that still gets the biology right. Now that #32537 has severed the amino acid metabolism link, this is what remains.

## Proposed final hierarchy

```
gene expression
└─ protein biosynthetic process                      GO:0160307
   │
   ├─has_part─ translation                           GO:0006412
   │            ├─ cytoplasmic translation           GO:0002181
   │            ├─ mitochondrial translation         GO:0032543
   │            └─ plastid translation               GO:0032544
   │
   └─has_part─ tRNA charging                         GO:0043039   [renamed]
                ├─ tRNA aminoacylation for protein translation          GO:0006418
                │   └─ ...for mitochondrial protein translation         GO:0070127
                └─ ...for nonribosomal peptide biosynthesis             GO:0043040  [see #30872]
```

`tRNA charging` also keeps `is_a tRNA metabolic process` (GO:0006399). More on that below.

**Obsoleted (42):** the 20 amino-acid-specific terms, the 20 mitochondrial ones, GO:1990762 `cytoplasmic alanyl-tRNA aminoacylation`, and GO:0070681 `glutaminyl-tRNAGln biosynthesis via transamidation`.

**New terms required: none.**

## The four calls

### 1. Keep `is_a tRNA metabolic process` on GO:0043039

This reverses the plan, which had it removed on the grounds that tRNA is not metabolized.

`tRNA metabolic process` is a defined class: *metabolic process and has_primary_input_or_output some tRNA*. Charging is `tRNA + amino acid + ATP -> aminoacyl-tRNA`. The tRNA goes in, and an aminoacyl-tRNA — still a tRNA — comes out. That fits the definition. Removing the assertion would not stop the term being a tRNA metabolic process; it would just stop it saying so, and would leave GO:0043039 with no genus at all.

It is also the mirror image of the argument that fixed this ticket. The amino acid is *attached to* something and is not itself transformed, so this is not amino acid metabolism. The tRNA *is* transformed, so it is tRNA metabolism. Same test, opposite answers, which is what makes it principled rather than convenient. Compare `tRNA modification` (GO:0006400), which sits under tRNA metabolic process on exactly this basis.

Add `part_of GO:0160307` alongside it. Dual parentage here is informative, not a defect.

### 2. Leave GO:0006418 and GO:0070127 alone

The sketch above needs a cytosolic sibling for GO:0070127, which would mean repurposing GO:0006418 to mean "cytosolic". That would silently change the meaning of 5,373 annotations and 15 InterPro2GO mappings without any ID changing — the worst kind of edit to review.

The asymmetry is real biology anyway: mitochondrial charging has its own enzymes, while "cytosolic" is the unmarked default. GO:1990762, the only cytoplasmic variant anyone ever created, has zero annotations.

### 3. No organelle-specific protein biosynthetic process terms

Compartment is already carried by the three translation terms and by GO:0070127. Three new groupers to hold pairs that are already located individually would add nodes without adding information.

@ValWood already made this point: *"we don't need a grouping term for translation, grouping these processes by the protein synthesis term would be sufficient"*. And @hattrill's question about `starts_with`/`ends_with` has no good answer — *"starts with is tricky, it includes amino acid charging and translation initiation but neither comes first"*. A term that resists definition is usually telling you something.

### 4. Obsolete the leaves, but replace them compartment-aware

All 40 are exact 1:1 mirrors of existing MF terms (`glycyl-tRNA aminoacylation` / `glycine-tRNA ligase activity`, and so on), so the MF_in_BP diagnosis holds. I checked whether anything is lost: for the three best-used terms, **28 of 28** gene products with experimental annotations already carry the paired MF annotation. The specificity is preserved.

The plan says every obsoleted term gets `replaced_by GO:0043039`. Better:

| Terms | `replaced_by` |
|---|---|
| 20 cytosolic leaves | **GO:0006418** |
| 20 mitochondrial leaves | **GO:0070127** |

Same obsoletion count, no extra work, and it keeps the cytosolic/mitochondrial distinction that collapsing everything to GO:0043039 would throw away.

## One call I am less sure of

**GO:0070681 glutaminyl-tRNAGln biosynthesis via transamidation** (770 annotations, 14 experimental). @ValWood flagged this as valid. Against that: its exact structural twin, GO:0070680 `asparaginyl-tRNAAsn biosynthesis via transamidation`, was already obsoleted in #31932 as *"a specific pathway variant, which is out of scope for GO"*. Keeping one and not the other is hard to defend, so I have it as an obsoletion — but if you would rather keep it, #31932 should be revisited too.

If it does go, please do not copy the twin's replacement. GO:0070680 was given `replaced_by GO:0070981 L-asparagine biosynthetic process`, which is wrong — the pathway produces Asn-tRNA(Asn), not free asparagine. GO:0006418 is the right target for GO:0070681.

## Cost comparison

| | This proposal | Organelle build-out |
|---|---|---|
| New terms | 0 | 3+ |
| Terms left without a genus | 0 | 1 |
| High-traffic terms redefined | 0 | 1 |
| Obsoletions | 42 | 45+ |

Net change to the surviving structure: one label swap and one added `part_of`. Everything else is leaf pruning with clean replacement targets.

## Suggested sequencing

1. **Now, no annotation impact:** rename GO:0043039 to `tRNA charging`, add `part_of GO:0160307`. Two lines.
2. **Then:** the 42 obsoletions with the replacement targets above, which is where @edwong57's 290 annotations get handled.
3. **Separately:** GO:0043040 / GO:0043041 / GO:0043042 on #30872. Worth noting two of the three MF terms that ticket is waiting on already exist — `phosphopantetheine-dependent carrier activity` (GO:0140414) for the T domain and `amino acid ligation activity by nonribosomal peptide synthase` (GO:0097429) for the C domain. Only the adenylation domain lacks one.

Happy to implement step 1 immediately if this looks right. Steps 2 and 3 want a curator's sign-off first, particularly on GO:0070681.

**Comment by raymond91125** [comment]
[Written by Claude]

#32541 is merged. Correcting one thing in my [recommendation above](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-5431742836), because the PR did not land as I proposed it.

## What is now in the ontology

- **GO:0043039 is renamed `tRNA charging`**, with `tRNA aminoacylation` kept as an EXACT synonym and `aminoacyl tRNA synthesis` added, per @pgaudet's annotation call.
- **`is_a tRNA metabolic process` is retained**, as argued.
- **`part_of GO:0160307` was NOT added.** I proposed it; review rejected it, correctly.

## Why the part_of was wrong

I had it as one of only two spine edits, so it is worth being explicit that it was my error rather than a deferral.

`part_of` on a parent is a universal claim: it says *every* tRNA charging event is part of protein biosynthesis. That is falsified by `GO:0043039`'s own subclass `GO:0043040 tRNA aminoacylation for nonribosomal peptide biosynthetic process`, which is `part_of GO:0019184`, sitting under `peptide biosynthetic process` rather than `protein biosynthetic process`. Nonribosomal peptides are not proteins, and `GO:0160307`'s equivalence axiom pins its `has_primary_output` to `PR:000000001 protein`. So the axiom would have asserted that NRP-directed charging is part of protein synthesis, which is false.

What makes this a plain miss on my part: I created that counterexample myself two commits earlier, when I reparented `GO:0043040` onto `part_of GO:0019184` in response to the previous round of review, and then asserted the universal `part_of` without checking it against my own change.

The rollup that motivated the axiom was already there without it:

```
GO:0006418 tRNA aminoacylation for protein translation
  part_of GO:0006412 translation
    part_of GO:0160307 protein biosynthetic process
```

Verified by closure over `is_a` and `part_of` on current master:

```
GO:0043039 tRNA charging                    -> GO:0160307 : NO   (correct - not universal)
GO:0006418 ...for protein translation       -> GO:0160307 : YES  (translation-directed rollup intact)
GO:0043040 ...for nonribosomal peptides     -> GO:0160307 : NO   (correct - NRPs are not protein)
```

`GO:0160307` also still carries the sound direction of the relation in its logical definition, `has_part GO:0043039` - protein synthesis has tRNA charging as a part, which is true and unaffected.

Review also removed a stale `"tRNA charging" NARROW` synonym from `GO:0006418` that the rename turned into a label/synonym collision with inverted scope. I had checked for stale label comments but not for pre-existing uses of the new label, which is how it got through.

## One consequence for the remaining plan

Item 4 of the tracked summary asks for `part_of GO:0160307` on the retained term. As things stand that item should be marked **not applicable** rather than outstanding.

It becomes applicable again only if `GO:0043040` is obsoleted through #30872. Remove the nonribosomal subclass and every remaining child of `tRNA charging` is translation-directed, at which point the universal `part_of` is true and could be added. Worth sequencing that way rather than revisiting it now.

## Still open, unchanged

The 43-term obsoletion with the compartment-aware replacement targets, and #30872. Both still want a curator's sign-off, particularly on GO:0070681.

Annotation-side review of the affected experimental annotations is done and is on geneontology/go-annotation#6525 - 204 of 251 rows already carry the paired MF from the same reference, and there are six annotations that should not move to a ligase MF at all.

**Comment by raymond91125** [comment]
[Written by Claude]

Closing. The announcement went out, no objections were received, and the obsoletion is complete. Summary of where things landed, for anyone arriving from an old link.

## What the ticket asked

@Antonialock opened this in March 2018 asking whether `GO:0006426 glycyl-tRNA aminoacylation` and `GO:0006544 glycine metabolic process` should be related. The answer was no, but the ontology said otherwise: the whole aminoacylation branch sat under `amino acid metabolic process` via `amino acid activation`, which is what made the question awkward to answer and what the ticket turned into.

## What is now in the ontology

```
gene expression
└─ protein biosynthetic process                      GO:0160307
   ├─has_part─ translation                           GO:0006412
   └─has_part─ tRNA charging                         GO:0043039   (also is_a tRNA metabolic process)
                ├─ tRNA aminoacylation for protein translation      GO:0006418
                │   └─ ...for mitochondrial protein translation     GO:0070127
                └─ ...for nonribosomal peptide biosynthesis         GO:0043040  → #30872
```

Delivered in three PRs:

- **#32537** — severed `tRNA aminoacylation` from `amino acid metabolic process` and obsoleted `GO:0043038 amino acid activation`. This is the fix for the original question. `GO:0043041` was re-homed directly under amino acid metabolism, where its adenylylation chemistry belongs, and `GO:0043040` was reparented so no tRNA charging term was left behind under amino acid metabolism.
- **#32541** — renamed `GO:0043039` to `tRNA charging` per @pgaudet's annotation call, with `tRNA aminoacylation` retained as an EXACT synonym. `is_a tRNA metabolic process` was **kept**: the reaction takes a tRNA in and yields an aminoacyl-tRNA, which satisfies that term's own definition. Review correctly rejected an attempt to add a universal `part_of GO:0160307`, since nonribosomal-peptide-directed charging is not part of protein synthesis; that rollup is already entailed through `GO:0006418 → translation → protein biosynthetic process`.
- **#32551** — obsoleted 43 amino-acid-specific terms, each of which restated an existing molecular function. `replaced_by` was split by compartment (22 → `GO:0006418`, 20 → `GO:0070127`) rather than collapsed onto `GO:0043039`, so the mitochondrial distinction survives. `GO:0070681` carries `consider` pointers only, with no `replaced_by`, because no single term is a safe automatic substitution for it.

@pgaudet's item (3), removing `protein metabolic process` and `macromolecule biosynthetic process` as asserted superclasses of `GO:0006412 translation`, is also done.

## Still open, with homes

- **#30872** — obsoleting `GO:0043041` and `GO:0043042`. Two of the three NRPS domain MFs that ticket is waiting on already exist: `GO:0140414 phosphopantetheine-dependent carrier activity` (T domain) and `GO:0097429 amino acid ligation activity by nonribosomal peptide synthase` (C domain). Only the adenylation domain needs a new MF term.
- **geneontology/go-annotation#6525** — annotation and mapping migration. Of 251 non-IEA annotations reviewed, 204 already carried the paired MF from the same reference. Six annotations should not move to a ligase MF at all and are flagged there for their groups. 170 mappings (InterPro2GO 63, UniRule 62, HAMAP 44, MetaCyc 1) still need retargeting; mapping contacts are assigned.

## One question this ticket raised and did not settle

@ValWood's organelle build-out — cytosolic / mitochondrial / plastid `protein biosynthetic process` grouping terms, and @pgaudet's item (2). My [recommendation](https://github.com/geneontology/go-ontology/issues/15375#issuecomment-5431742836) was not to mint them, on the grounds that compartment is already carried by the three translation terms and by `GO:0070127`, and that the `starts_with`/`ends_with` definition @hattrill asked about has no good answer. Nobody ruled on that either way. It is not blocking anything, but if it should be pursued it needs its own ticket rather than living on here.
